<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>helpWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>127</width>
      <height>80</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>196</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="upButton">
         <property name="text">
          <string>up</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="downButton">
         <property name="text">
          <string>down</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="leftButton">
         <property name="text">
          <string>left</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="rightButton">
         <property name="text">
          <string>right</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="enterButton">
       <property name="text">
        <string>enter</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actWidgetInsite"/>
   <addaction name="actQuit"/>
  </widget>
  <action name="actWindowInsite">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/2.ico</normaloff>:/images/images/2.ico</iconset>
   </property>
   <property name="text">
    <string>嵌入式MainWindow</string>
   </property>
   <property name="toolTip">
    <string>嵌入式MainWindow</string>
   </property>
  </action>
  <action name="actWidgetInsite">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/100.bmp</normaloff>:/images/images/100.bmp</iconset>
   </property>
   <property name="text">
    <string>helpWidget</string>
   </property>
   <property name="toolTip">
    <string>Widget嵌入式窗体</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/132.bmp</normaloff>:/images/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出本系统</string>
   </property>
  </action>
  <action name="actDoc_Open">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/122.bmp</normaloff>:/images/images/122.bmp</iconset>
   </property>
   <property name="text">
    <string>打开文档</string>
   </property>
   <property name="toolTip">
    <string>打开文档</string>
   </property>
  </action>
  <action name="actWindow">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/3.ico</normaloff>:/images/images/3.ico</iconset>
   </property>
   <property name="text">
    <string>独立MainWindow窗口</string>
   </property>
   <property name="toolTip">
    <string>独立MainWindow窗口</string>
   </property>
  </action>
  <action name="actWidget">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/804.bmp</normaloff>:/images/images/804.bmp</iconset>
   </property>
   <property name="text">
    <string>独立Widget窗口</string>
   </property>
   <property name="toolTip">
    <string>新建Widget独立窗口</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
